crates/vm/levm/src/opcode_handlers/exchange.rs

```diff
@@ -13,9 +13,6 @@ impl<'a> VM<'a> {
         let current_call_frame = &mut self.current_call_frame;
         current_call_frame.increase_consumed_gas(gas_cost::SWAPN)?;
 
-        if current_call_frame.stack.len() < N {
-            return Err(ExceptionalHalt::StackUnderflow.into());
-        }
```

Comment:
This check is different from the one in `swap`, but actually wrong, since a stack length of 1 and an N of 1 would pass this check and fail inside `Stack::swap`. Writing this here so no one else goes down the rabbit hole 😞